src/main/java/com/fasterxml/jackson/databind/deser/BeanDeserializerBase.java

```diff
@@ -645,7 +645,7 @@ public void resolve(DeserializationContext ctxt) throws JsonMappingException
         }
 
         // And now that we know CreatorProperty instances are also resolved can finally create the creator:
-        if (creatorProps != null) {
+        if (creatorProps != null && creatorProps.length > 0) {
```

Comment:
So the idea is... it should not have `create()` method as PropertyBasedCreator, but instead make it default creator for Builder case.

Reply:
I do not follow the logic here. Why should we skip the case of 0-arguments Creator?  (this would only apply to factory method, I think, with constructor we get separate "default constructor" case). It also does not seem to apply just to the Builder case (no checks to indicate that).